scripts/db_connect.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# This script requires google's cloud_sql_proxy on your PATH. One way to set this up:
+# $ brew cask install google-cloud-sdk
+# $ gcloud components install cloud_sql_proxy
+# $ PATH=$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/
+# OR
+# $ ln -s /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/cloud_sql_proxy /usr/local/bin/cloud_sql_proxy
+#
+# Use this script to connect to cloud Postgres for a specific Catalog instance.
+# For example, to connect to the dev catalog database, run:
+# $ ENV=dev ./db-connect.sh
+#
+# The proxy will continue to run until you quit it using ^C.
+#
+# The default port used is 5431, to avoid conflicting with a locally running postgres which
+# defaults to port 5432. Use the environment variable PORT to override this setting.
+
+: "${ENV:?}"
+PORT=${PORT:-5431}
+
+VAULT_PATH=secret/dsde/terra/kernel/${ENV}/${ENV}/catalog/postgres
+
+INSTANCE=$(vault read -field=data -format=json "${VAULT_PATH}/instance" |
+           jq -r '"\(.project):\(.region):\(.name)"')
+
+JDBC_URL=jdbc:postgresql://localhost:$PORT/$(vault read -field=data -format=json "${VAULT_PATH}/db-creds" |
+          jq -r '"\(.db)?user=\(.username)&password=\(.password)"')
```

Comment:
I don't think we can be sure that a `jdbc` connection is set up appropriately in Intellij or another tool, especially since some devs are using the community edition which I believe does not support database connections. Instead I would output the url (without jdbc) and perhaps also automatically connect through `psql`. All team members should have `psql` installed as part of the Postgres.app setup in our getting started docs. This would allow devs to have a default starting point if desired, but allow them to use other tools also.

Reply:
This code formats a JDBC URL and prints it to stdout so a user can cut & paste this and use it in IntelliJ or any other tool that supports JDBC URLs. I could remove the `jdbc` prefix, but without it, what tool would be able to use the URL?

I'll add code that emits the `psql` command as well as the URL, so users can pick the one that matches what they need.

Reply:
Oh, I didn't realize the psql command line accepted the `postgresql` link as an argument. I agree, I can make the code emit this instead.